Tests/HTMLEntitiesTests/HTMLEntitiesTests.swift

```diff
@@ -398,6 +402,52 @@ class HTMLEntitiesTests: XCTestCase {
         XCTAssertEqual(try text.htmlUnescape(strict: false), "한")
     }
 
+    func testDecodeMaps() throws {
+        struct CodePointsAndCharacters: Codable {
+            var codepoints: [UInt32]
+            var characters: String
+        }
+
+        var entitiesData: Data?
+        var dataTaskError: Error?
+        let expectation = self.expectation(description: "Downloading entities.json")
+
+        let url = URL(string: "https://html.spec.whatwg.org/entities.json")!
+        URLSession.shared.dataTask(with: url) { data, _, error in
+            if let error = error {
+                dataTaskError = error
+            } else if let data = data {
+                entitiesData = data
+            }
+            expectation.fulfill()
+        }.resume()
+
+        self.wait(for: [expectation], timeout: 60)
```

Comment:
Nice touch adding a test against a known reference.  I'm not sure it's a good idea to hit the server on every unit test, though.  There could be other problems (network issues, unexpected changes upstream etc.).  Perhaps it'd be better to keep this in the repo?

At 140k raw, it's not "huge".  Assuming it's not something that changes often, it's probably fine to keep in the repo.

Reply:
Thank you for reviewing.

Your concerns are reasonable. I also thought about it at first. However, I realized that I would have to verify that the file was correct. It seems to me that using a local file to validate `*DecodeMap`s simply postpones the problem, since another way is needed to validate the file itself.

Moreover, since the HTML specs keep changing, there will come a time when the file will have to be changed. We must then verify in some way that those changes are correct.

What do you think?

Reply:
That's actually added reason to keep a copy of the file in the repo so that it can be manually inspected each time.  If it's automated, then the test can break at any time and it'll be unclear why.

Reply:
Thanks for replying. The main point I wrote was that it didn't make sense to add a file to the repo because I thought verifying it wasn't much different than verifying Swift constants. But it's a copy of a file published as a part of the specs, so it is easier to verify than Swift constants. I changed my mind. I'm going to add the file and rewrite the test.

Reply:
I pushed the commit b86aa375c18ee6e5811a7c5b91a8bd93ae126b86.

Reply:
@dannys42 I am not in a hurry, but what do you think?